app/controllers/post_controller.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+from flask import request, Blueprint, render_template, redirect, url_for, flash
+from app.services import post_service
+from app.utils.file import FORM_POST_IMAGE
+from app.utils.session_decorators import session_required
+from app.utils.validation import is_post_image_valid
+
+post_routes = Blueprint('post_routes', __name__)
+
+
+@post_routes.route('/post', methods=['GET', 'POST'])
+@session_required
+def create_post(user_id: int) -> Any:
+    if request.method == 'GET':
```

Comment:
Will need to be refactored similar to `auth_controller.sign_up()` once that Flask form is implemented. Should probably add a comment so whoever is working on this section in the front end is aware when errors start showing up.